### ISSUE TITLE
Shell completion

### ISSUE THREAD

**Issue by Narrat** [archlinux32/asp32#1]
At least with the zsh completion file there is a problem.
The compdef line needs to adjusted.
Else the completion won't be used with asp32 and when typing asp (originally binary name) and tabbing it will error out.

**Comment by deep-42-thought** [comment]
do you mind to provide a patch/pull request? I'm quite unfamiliar with zsh and shell-completion in general

**Comment by Narrat** [comment]
Not that I know that much more of creating completion files (although working on it for a different program).
But I can at least do that and will shortly provide a PR

Edit: Need to look at this further. Different machines, different outcome. Creating an asp32 package for my AL machine, the completion file is found.
On the AL32 it fails and does an asp completion instead. Readjusting the compdef line didn't help in that case.

**Comment by deep-42-thought** [comment]
ah, that might be due to our package-renaming: `asp` on archlinux32 is `asp32` on archlinux (x86_64)

**Comment by Narrat** [comment]
Yeah, but that doesn't change the content of the files (with everything renamed to `asp32`). $pkgname is irrelevant.
Kinda weird currently, but I'm still investigating it

**Comment by Narrat** [comment]
Sorry, got busy and kinda forgot about this.
Problem is partially solved. Partially because adjusting the the compdef line does prevent the completion on asp, as I initially thought. Testing this before I didn't reload the completion file the right way *cough*
Still no PR, as the error is gone, but no completion happens whatsoever. Dunno what Shell magic is there at work and need to look into that. But debugging shell scripts isn't my strong side, so this may take some time.
